samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -20,7 +20,7 @@ static void start_scan(void);
 static struct bt_conn *default_conn;
 static struct k_work_delayable audio_send_work;
 static struct bt_audio_unicast_group *unicast_group;
-static struct bt_codec *remote_codec_capabilities[CONFIG_BT_AUDIO_UNICAST_CLIENT_PAC_COUNT];
+static struct bt_codec remote_codec_capabilities[4];
```

Comment:
If this is assuming 4 to match the unicast_audio_server, maybe it should be mentioned in a comment?

Reply:
It is not defined to match the unicast audio server - It is just an arbitrary value. 

We could consider adding it as a Kconfig for the sample.